qiskit/utils/measurement_error_mitigation.py

```diff
@@ -209,19 +225,6 @@ def build_measurement_error_mitigation_qobj(
         QiskitError: when the fitter_cls is not recognizable.
         MissingOptionalLibraryError: Qiskit-Ignis not installed
     """
-    try:
-        from qiskit.ignis.mitigation.measurement import (
-            complete_meas_cal,
-            tensored_meas_cal,
-            CompleteMeasFitter,
-            TensoredMeasFitter,
-        )
-    except ImportError as ex:
-        raise MissingOptionalLibraryError(
-            libname="qiskit-ignis",
-            name="build_measurement_error_mitigation_qobj",
-            pip_install="pip install qiskit-ignis",
-        ) from ex
 
```

Comment:
it's not strictly necessary because the calibration circuits are the same. Above where we handle the ignis path we're actually just passing the terra version of the classes here.

If ignis were going to continue active feature development I would have added handling for calling ignis here just in case the cal circuits diverged, but since everything is pretty much froze and this is just for compat while the api for mitigation in algorithms is being redone I think this is fine.